superset/assets/.eslintrc

```diff
@@ -26,7 +27,7 @@
     "no-mixed-operators": 0,
     "no-continue": 0,
     "no-bitwise": 0,
-    "no-undef": 0,
+    "no-undef": 1,
```

Comment:
warning for this, turning this off is BAD!

Reply:
+100

Reply:
This one total worths its own PR.